Modules/_functoolsmodule.c

```diff
@@ -1413,7 +1413,9 @@ static PyMethodDef _functools_methods[] = {
 static void
 _functools_free(void *m)
 {
-    Py_CLEAR(kwd_mark);
+    // FIXME: Avoid double free when using multi-phase initialization (PEP-489)
+    //        Will fix when PEP-573 gets in.
```

Comment:
The comment is not correct. The bug is not "double free": Py_CLEAR(NULL) does NULL, that's fine. The problem is if one module instance is deallocated, kwd_mark is cleared to NULL. Other instances will use NULL and so crash.

Reply:
Rephrased the comment and updated commit message. Thanks for getting it.